Small QE updates (#737)
opendatahub-io/opendatahub-documentation: modules/lmeval-scenarios.adoc

.Verification
. After deploying the `LMEvalJob`, open the `kubectl` command-line and enter this command to check its status: `kubectl logs -n test job/evaljob-sample -n test`
. View the logs with the `kubectl` command `kubectl logs -n test job/<job-name>` to make sure it has functioned correctly. 

> . The results will be displayed in the logs once evaluation completes.

Reviewer: ```suggestion
. The results are displayed in the logs after the evaluation is completed.
```
https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense - "Write in the simple present tense as much as possible."
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#o - "Use only to mean one time. Do not use as a conjunction to mean after or when."
